### PR TITLE
WD: Salary survey project

### PR DESCRIPTION
**Pull request by goatlady**
----
📚 Documentation preview 📚: https://writethedocs-www--2123.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2024/writing-day.rst [hunk 1]
```diff
@@ -164,6 +164,19 @@ Our Writing Day goal is to onboard contributors and improve the Museum API defin
 
 The Museum API was built specifically to teach OpenAPI concepts and to create a better developer experience. The more realistic and complex the Museum API is, the better it can demonstrate all that OpenAPI has to offer.
 
+Write the Docs Documentation Salary Survey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Kay Smoljak, she/her
+
+The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as spark discussion on other topics such as remote work, pay transparency, and job satisfaction.
```
Comment (plaindocs) [inline docs/conf/portland/2024/writing-day.rst]:
```suggestion
The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as sparking discussion on other topics such as remote work, pay transparency, and job satisfaction.   
```

File: docs/conf/portland/2024/writing-day.rst [hunk 2]
```diff
@@ -164,6 +164,19 @@ Our Writing Day goal is to onboard contributors and improve the Museum API defin
...
 The Museum API was built specifically to teach OpenAPI concepts and to create a better developer experience. The more realistic and complex the Museum API is, the better it can demonstrate all that OpenAPI has to offer.
 
+Write the Docs Documentation Salary Survey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Kay Smoljak, she/her
+
+The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as spark discussion on other topics such as remote work, pay transparency, and job satisfaction.   
+ 
+The format, questions and instructions have evolved each year, based on the submitted responses and feedback from the community, but it's largely been a one-person effort. The 2023 results - which will be available before Writing Day - are drawn from the largest respondent pool yet, and for the first time include structured as well as freeform feedback on the survey itself.
```
Comment (plaindocs) [inline docs/conf/portland/2024/writing-day.rst]: Maybe reword `The format, questions and instructions` ?

Do you mean `The format, questions, and instructions` or

**Review by plaindocs — APPROVED** [review]
Looks great

File: docs/conf/portland/2024/writing-day.rst [hunk 3]
```diff
@@ -156,14 +156,39 @@ Already familiar with GitBook? We're looking for people like you to brainstorm f
...
+The Museum API is a sample OpenAPI description. It offers a set of endpoints to interact with a museum's services -- such as retrieving museum hours, managing special events, and purchasing tickets.
 
 Our Writing Day goal is to onboard contributors and improve the Museum API definition. Help us create more complex use case examples like discriminator, webhooks, oneOf, allOf, etc. 
 
 The Museum API was built specifically to teach OpenAPI concepts and to create a better developer experience. The more realistic and complex the Museum API is, the better it can demonstrate all that OpenAPI has to offer.
 
+Write the Docs Documentation Salary Survey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Kay Smoljak, she/her. Afternoon session.
```
Comment (rosewms) [inline docs/conf/portland/2024/writing-day.rst:170]: @goatlady and @plaindocs - I edited this for style and format as well as some content reordering. All the content was awesome so my goal was to make it friendlier to folks scanning the Project List.

**Review by plaindocs — APPROVED** [review]
Looks good @rosewms , hit it! 

(there is a bonus typo in the Open API section above)

**Comment by rosewms** [comment]
@plaindocs I'll get in the next project PR that I'm about to draft.
